database/service_db.go

```diff
@@ -64,7 +65,8 @@ func (db *ServiceDB) unmarshal(id string, value []byte) (*service.Service, error
 }
 
 // All returns every service in database.
-func (db *ServiceDB) All() ([]*service.Service, error) {
+// If ids > 0, it'll only return services with matching ids.
```

Comment:
It's better to keep the functionality of `All` as it's was by original.

You can add an GetByIDs or similar method, to keep API more transparent